deeprank/generate/DataGenerator.py

```diff
@@ -331,7 +335,7 @@ def create_database(
                 variant_group.require_group('grid_points')
 
                 try:
-                    center = self._get_grid_center(variant)
+                    center = DataGenerator.get_grid_center(self.environment, variant)
```

Comment:
The tests are failing.
It could be due to this part. May be self.environment isn't necessary.

Reply:
The tests are failing, because on github, pytest somehow cannot find the included PDB test files. I'm working on it..

The self.environment holds the path to the directory where PDB is located. In this particular branch, variant objects don't hold pdb paths, but just accession codes. Do you want to keep it like this, or switch back to having the path in the variant object ?

Reply:
the self.environment can hold the pdb path.